task.css

```diff
@@ -145,18 +131,20 @@ body{
 
 
 .footer{
-    
     background-color: rgb(34, 28, 88);
     height: 35px;
     width: 100%;
     margin:0px;
```

Comment:
I generally avoid giving units with 0 values; 
I mean margin:0 would be sufficients here.